CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- A new `--quiet` flag to suppress noisy output in scripting scenarios (#437)
+
```

Comment:
Please, remove the changelog changes, they are automatically generated from the commit titles (please, update the PR title to be "feat: A new `--quiet` flag to suppress noisy output in scripting scenarios")